src/pages/DirectMessage.tsx

```diff
@@ -226,9 +227,9 @@ const DirectMessage = () => {
                       </Box>
                     }
                     secondary={
-                      <Typography sx={styles.timestamp} variant="body2">
-                        {formatTimestamp(o.timestamp)}
-                      </Typography>
+                      <Box>
```

Comment:
Does this `Box` need to be here?

Reply:
Nope, It was needed when I was initially working on it. Fixed it by removing it